workflows/dev-requirements.txt

```diff
@@ -1,10 +1,5 @@
 flake8~=3.7
-ipython~=8.10
-jedi~=0.15.0
-black~=24.3.0
 pytest~=5.3
 pytest-watch~=4.2
-awscli~=1.27.90
-boto3~=1.11
 setuptools>=65.5.1 # not directly required, pinned by Snyk to avoid a vulnerability
 rsa>=4.7 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
We could probably get rid of `setuptools` and `rsa` at this point. Not even sure what would be using `rsa`

Reply:
It looks like pytest-watch runs the linter and the tests too. Atleast according to the readme